tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java

```diff
@@ -126,6 +127,24 @@ public void testAddTenantSucceedsWithAGeneratedId(final VertxTestContext context
             }));
     }
 
+    /**
+     * Verifies that a request to create a tenant with a body that exceeds the registry's max payload limit
+     * fails with a {@link HttpURLConnection#HTTP_ENTITY_TOO_LARGE} status code.
+     *
+     * @param context The vert.x test context.
+     */
+    @Test
+    public void testAddTenantFailsForRequestPayloadExceedingLimit(final VertxTestContext context)  {
```

Comment:
How about a test also to check for limit during _update Tenant_ operation?

Reply:
sure, I'll add one

Reply:
I have pushed a change ...

Reply:
@kaniyan Would you mind taking another look?

Reply:
@sophokles73 There are file conflicts. Can you please rebase?

Reply:
done